src/Playwright.Tests/TestConstants.cs

```diff
@@ -33,17 +33,36 @@ namespace Microsoft.Playwright.Tests
 {
     internal static class TestConstants
     {
-        public static string BrowserName => NUnit.PlaywrightTest.BrowserName;
+        public static string BrowserName { get; set; } = null!;
 
         public const int DefaultTestTimeout = 30_000;
         public const int DefaultTimeout = 10_000;
         public const int DefaultTaskTimeout = 5_000;
 
-        public const string AboutBlank = "about:blank";
+        internal static bool IsWebKit
+        {
+            get
+            {
+                return BrowserName == BrowserType.Webkit;
+            }
+        }
+
+        internal static bool IsFirefox
+        {
+            get
+            {
+                return BrowserName == BrowserType.Firefox;
+            }
+        }
+
+        internal static bool IsChromium
+        {
+            get
+            {
+                return BrowserName == BrowserType.Chromium;
+            }
+        }
 
-        internal static readonly bool IsWebKit = BrowserName == BrowserType.Webkit;
```

Comment:
nit: Why did it change? The previous code was more concise.

Reply:
BrowserName gets set lazily, so we need a getter which gets executed.

This is only for our internal tests too.